src/js/components/Calendar/Calendar.js

```diff
@@ -627,16 +640,9 @@ const Calendar = forwardRef(
             ) : (
               // theme.calendar.heading.level should be removed in v3 of grommet
               // theme.calendar[size].title should be used instead
```

Comment:
Could we move this comment up to line 268 so it is closer to where we are using `theme.calendar.heading.level`

Reply:
done